ote/resources/public/language/en.edn

```diff
@@ -690,6 +690,17 @@
           "charging-points" "Kuvaus latauspisteistä"
           "parking-capacities" "Parkkipaikkojen määrä"
 
+          "available-from" "Available from"
+          "available-to" "Available to"
+
```

Comment:
How are  :ote.db.transport-service/available-from-nil "Heti" :ote.db.transport-service/available-to-nil "Toistaiseksi" presented in the geojson?

Reply:
They are not presented at all.